recipes/sleef/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set version = "3.5.1" %}
+
+package:
+  name: sleef
+  version: {{ version }}
+
+source:
+  url: https://github.com/shibatch/sleef/archive/{{ version }}.tar.gz
+  sha256: 415ee9b1bcc5816989d3d4d92afd0cd3f9ee89cbd5a33eb008e69751e40438ab
+  patches:
+    - install_dll.diff  # [win]
+
+build:
+  number: 0
+  run_exports:
+    # SO name changes at minor rev bumps.
+    - {{ pin_subpackage('sleef', max_pin='x.x') }}
```

Isn't the SONAME `libsleef.so.3`?
```suggestion
    - {{ pin_subpackage('sleef', max_pin='x') }}
```

---

It is, but I am not 100% sure what their ABI stability policy is. I thought I was just being overly conservative via an ``x.x`` pin. Does the ``x`` pin mean that version 3.n is considered compatible with 3.m?